purchase_order_weight_volume/models/purchase_order.py

```diff
@@ -50,6 +50,25 @@ class PurchaseOrder(models.Model):
         "Display Volume in Report", default=True
     )
 
+    display_order_weight_in_po = fields.Boolean(
+        "Display Order Weight in PO",
+        compute="_compute_display_order",
+    )
+    display_order_volume_in_po = fields.Boolean(
+        "Display Order Volume in PO",
+        compute="_compute_display_order",
+    )
+
+    @api.depends("company_id")
```

Comment:
AFAIK, @api.depends_context("company_id") should be used instead of @api.depends :thinking:

Reply:
My understanding is that when the user.company changes `_compute_display_order` is triggered with `@api.depends_context`
But company id is a stored field. So I think is okay to use  `@api.depends`